tfidf_logreg.py

```diff
@@ -0,0 +1,38 @@
+# tfidf_logreg.py - Performs logistic regression on the phrase/sentiments with k-fold
+#                   cross-validation, and prints one-vs-one and one-vs-all
+#                   validation accuracy.
+#
+# Usage: python -B tfidf_logreg.py
+#
+# Wesley Chavez 08-27-18
+
+import config
+import pandas as pd
+import numpy as np
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.linear_model import LogisticRegression
+from sklearn.multiclass import OneVsRestClassifier
+from sklearn.multiclass import OneVsOneClassifier
+from sklearn.model_selection import cross_val_score
+
+def main():
+    train = pd.read_csv('~/Downloads/train.tsv', sep="\t")
+    test = pd.read_csv('~/Downloads/test.tsv', sep="\t")
+    tfidf = TfidfVectorizer()
+
+    x_train = tfidf.fit_transform(train['Phrase'])
+    print (x_train.shape)
+    y = train['Sentiment']
+
+    logreg = LogisticRegression()
```

Comment:
What are testing here? because I don't have the dataset it's hard to tell. A comment describing the task, the expected input in train['Phrase'] and the categories for train['Sentiment'] would help. 

Perhaps discuss a little (in comments) why you think starting with logistic regression + ovr and logreg + ovo is a reasonable baseline?

Reply:
LR's a good baseline because it's simple, statistical, and quick to train.  I have no intuition on whether ovr or ovo performs better, so I added them as the beginning of a grid search.